npm/package.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "@bazel/ibazel",
+    "version": "0.0.1",
```

Comment:
how will this number be updated as we tag versions in git?

Reply:
Right now I don't have any git tags but it will be easy to sed it to the git tag when we do that. NPM will prevent publishing multiple versions with the same number so I believe this hardcoded is safe for the moment

Reply:
I changed it to v0.0.1 so that it will match what I intend to tag in git with